src/mono/mono/metadata/assembly-load-context.c

```diff
@@ -180,6 +193,18 @@ mono_alc_assemblies_unlock (MonoAssemblyLoadContext *alc)
 	mono_coop_mutex_unlock (&alc->assemblies_lock);
 }
 
+void
+mono_alc_memory_managers_lock (MonoAssemblyLoadContext *alc)
+{
```

Comment:
Is this used/going to be used outside this file ?

Reply:
Yes, it gets used from memory-manager.c